config/styles/blade-effects.md

```diff
@@ -31,25 +40,51 @@ Never generated, used as defaults in some places to mean "no effect", but is oth
 ### EFFECT_POSTOFF  
 ### EFFECT_IGNITION  
 ### EFFECT_RETRACTION  
-
 ### EFFECT_CHANGE  
 When using color change.  
+### EFFECT_FAST_ON  
+When using a feature that skips preon.
+### EFFECT_FAST_OFF  *(OS 7.0+)*
+When using a feature that bypasses postoff.    
+### EFFECT_SECONDARY_IGNITION  *(OS 7.0+)*  
+### EFFECT_SECONDARY_RETRACTION  *(OS 7.0+)*
+### EFFECT_OFF  *(OS 7.0+)*
+### EFFECT_OFF_CLASH  *(OS 7.0+)*
+### EFFECT_INTERACTIVE_PREON  *(OS 7.0+)*
+### EFFECT_INTERACTIVE_BLAST  *(OS 7.0+)*
+### EFFECT_BEGIN_AUTO_BLAST  *(OS 7.0+)*
+### EFFECT_END_AUTO_BLAST  *(OS 7.0+)*
+
+## Sound Events
+
+### EFFECT_VOLUME_LEVEL  *(OS 7.0+)*
```

Comment:
I think when you put things on the ### line, they become part of the link anchor, which I don't think we want.

Reply:
It is exactly as I found it. I guess I will try to see the navs available.